### PR TITLE
[Excel PowerPoint] (ui) animated gif for contextual tabs and enabling…

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
… buttons

### CODE REVIEW

**Comment by lindalu-MSFT** [comment]
Thanks Ricky, illlook into this tomorrow if that’s ok!

Thanks, Linda

**Please ignore random typos. I may have dictated this message.**

________________________________

**Comment by ElizabethSamuel-MSFT** [comment]
@Rick-Kirkham I merged from master into your branch so the diff should be better now.

File: docs/includes/animation-contextual-tabs-enable-button.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+The following animation shows how events can toggle the visibility of contextual tabs and change the enabled status of buttons.
+
+1. At the start, focus is not on the table.
+1. A table cell is clicked and the custom contextual tab **Table Data** appears on the ribbon (in addition to the built-in contextual tab **Table Design**).
+1. Selecting the **Table Data** tab opens it. Two of the buttons on the tab are disabled.
+1. Changing data in a cell enables the two buttons.
+1. Clicking one of the buttons, disables the two buttons again.
```
Comment (lindalu-MSFT) [inline docs/includes/animation-contextual-tabs-enable-button.md:8]:
```suggestion
1. Clicking one of the buttons disables the two buttons again.
```

File: docs/includes/animation-contextual-tabs-enable-button.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+1. Clicking a cell outside the table, removes focus from the table and the two contextual tabs are removed from the ribbon.
```
Comment (lindalu-MSFT) [inline docs/includes/animation-contextual-tabs-enable-button.md:9]:
```suggestion
1. Clicking a cell outside the table removes focus from the table, and the two contextual tabs are removed from the ribbon.
```

**Review by lindalu-MSFT — APPROVED** [review]
Hey Ricky, just a couple errant commas. Otherwise, good to go!

**Comment by Rick-Kirkham** [comment]
Thanks. That worked. But I’m surprised that’s necessary. I had already done that on the client.

**Review by alison-mk — APPROVED** [review]

**Pull request merged**